woocommerce/woocommerce
# Fix link click for shopper.searchForProduct

### All Submissions:






### Changes proposed in this Pull Request:


When using the `@woocomerce/e2e-utils` package, the `shopper.searchForProduct()` function is hanging when trying to click on a product name. It's clicking on the other link container rather than the actual link. This fixes it.

### How to test the changes in this Pull Request:

1. Create a simple e2e test that uses `shopper.searchForProducts()` function to search for a product
2. The test should click through to the product page for the given product you searched for.

### Other information:

* [ ] (n/a) Have you written new tests for your changes, as applicable?
* [ ] (n/a) Have you successfully run tests with your changes locally?



### Changelog entry

> Fix a minor issue when searchign for products in the e2e-utils package

### FOR PR REVIEWER ONLY:
